### ISSUE TITLE
Unable to use without babel-preset-es2015

### ISSUE THREAD

**Issue by vkbansal** [dashed/shallowequal#11]
Getting the following error
```
Error: Couldn't find preset "es2015" relative to directory "/Users/<username>/Projects/<project-name>/node_modules/shallowequal"
```

I'm using `babel-preset-env` and not `es2015`

**Comment by dashed** [comment]
Hey, @vkbansal , can I get some more information on how you're using shallowequal? is it in webpack?

**Comment by vkbansal** [comment]
Yes, I'm using webpack.

My build worked when I deleted `babel` key from `package.json`

**Comment by dashed** [comment]
@vkbansal I'm unable to re-produce this issue... 

I tried to create a quick app using create-react-app and added shallowequal as a dependency. And ran `yarn start` without issues.

**Comment by vkbansal** [comment]
I'll try to create an reproducible project and update you

**Comment by dashed** [comment]
Thanks that'd be helpful 👍 

I'm genuinely interested to see if my babel config is incorrect (or needs fixing) since it's only meant for tests.

**Comment by vkbansal** [comment]
Here you go: https://github.com/vkbansal/se-test

 - Clone it and cd
 - run `npm install` and `npm start`.

**Comment by dashed** [comment]
@vkbansal so I did some investigating. It appears you would need to do the following changes in your webpack (i.e. change from `'node_modules'` to `/node_modules/` in `exclude`):

```diff
diff --git a/webpack.config.js b/webpack.config.js
index 0a862b1..b6f59dd 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -24,6 +24,7 @@ let config = {
                 use: [{
                     loader: 'babel-loader',
                     options: {
+                        // babelrc: false,
                         presets: [
                             ['env', {
                                 targets: {
@@ -35,7 +36,7 @@ let config = {
                     }
                 }],
                 exclude: [
-                    'node_modules'
+                    /node_modules/
                 ]
             }
         ]
```

I haven't used string-based conditions before in webpack (only regex ones): https://webpack.js.org/configuration/module/#condition https://webpack.js.org/configuration/module/#rule-exclude

I also tried `babelrc: false`; which works. 

In addition, I tried whitelist approach (only include the `src` directory); which also works: https://webpack.js.org/configuration/module/#rule-include

**Comment by vkbansal** [comment]
Thanks. I learnt something new today.

**Comment by cooperka** [comment]
I also get this issue using shallowequal v1.0.1 with React Native. The React Native packager tries to consume all babel configs by default, and unfortunately it can't be turned off (as suggested above with webpack). If you were to use a `.babelrc` file (without publishing it), that would resolve this issue. I'll try to make a PR soon!

**Comment by dashed** [comment]
@cooperka oh interesting! 

I'll go ahead and merge your PR and cut a release soon.

**Comment by dashed** [comment]
@cooperka I've released shallowequal with your PR at `v1.0.2`.

**Comment by cooperka** [comment]
So quick! Thanks a lot @dashed.
